truck_test_rapid_analyzer.py

```diff
@@ -0,0 +1,614 @@
+    DESCENT_RATE_FT_S = "Constants!" + abs_coords(L_DESCENT_RATE_FT_S)
+
+    L_DESCENT_RATE_M_S = "C3"
+    DESCENT_RATE_M_S = "Constants!" + abs_coords(L_DESCENT_RATE_M_S)
+
+
+    L_ROCKET_MASS_TITLE = 'A4'
+
+    L_ROCKET_MASS_LB = "B4"
+    ROCKET_MASS_LB = "Constants!" + abs_coords(L_ROCKET_MASS_LB)
+
+    L_ROCKET_MASS_KG = "C4"
+    ROCKET_MASS_KG = "Constants!" + abs_coords(L_ROCKET_MASS_KG)
+
+
+    L_NOM_SA_TITLE = 'A5'
+
+    L_NOM_SA_M2 = "B5"
+    NOM_SA_M2 = "Constants!" + abs_coords(L_NOM_SA_M2)
+
+    L_NOM_SA_FT2 = "C5"
+    NOM_SA_FT2 = "Constants!" + abs_coords(L_NOM_SA_FT2)
+
+
+    L_TARGET_DRAG_AREA_TITLE = "A6"
+
+    L_TARGET_DRAG_AREA_M2 = "B6"
+    TARGET_DRAG_AREA_M2 = "Constants!" + abs_coords(L_TARGET_DRAG_AREA_M2)
+
+    L_TARGET_DRAG_AREA_FT2 = "C6"
+    TARGET_DRAG_AREA_FT2 = "Constants!" + abs_coords(L_TARGET_DRAG_AREA_FT2)
+
+
+    L_ANEMOMETER_FACTOR_TITLE = "A7"
+
+    L_ANEMOMETER_FACTOR = "B7"
+    ANEMOMETER_FACTOR = "Constants!" + abs_coords(L_ANEMOMETER_FACTOR)
+
+
+    L_LOAD_CELL_FACTOR_TITLE = 'A8'
+
+    L_LOAD_CELL_FACTOR = 'B8'
+    LOAD_CELL_FACTOR = "Constants!" + abs_coords(L_LOAD_CELL_FACTOR)
+
+
+    L_TARGET_COEFF_DRAG_TITLE = 'A9'
+
+    L_TARGET_COEFF_DRAG = 'B9'
+    TARGET_COEFF_DRAG = "Constants!" + abs_coords(L_TARGET_COEFF_DRAG)
+
+    def create_defined_names(self, target_workbook):
+        """
+        Turn the constants into excel defined names.
+        
+        Parameters
+        ----------
+            target_workbook: openpyxl.Workbook
+                The workbook where the defined names are the be created.
+        """
+        names = target_workbook.defined_names # Used to shorten lines 
+        
+        names.append(DName('NOM_DIAM_M', attr_text=self.NOM_DIAM_M))
+        names.append(DName('NOM_DIAM_FT', attr_text=self.NOM_DIAM_FT))
+
+        names.append(DName('AIR_DENSITY_KG_M3', attr_text=self.AIR_DENSITY_KG_M3))
+        names.append(DName('AIR_DENSITY_SLG_FT3', attr_text=self.AIR_DENSITY_SLG_FT3))
+
+        names.append(DName('DESCENT_RATE_M_S', attr_text=self.DESCENT_RATE_M_S))
+        names.append(DName('DESCENT_RATE_FT_S', attr_text=self.DESCENT_RATE_FT_S))
+
+        names.append(DName('ROCKET_MASS_LB', attr_text=self.ROCKET_MASS_LB))
+        names.append(DName('ROCKET_MASS_KG', attr_text=self.ROCKET_MASS_KG))
+
+        names.append(DName('TARGET_DRAG_AREA_M2', attr_text=self.TARGET_DRAG_AREA_M2))
+        names.append(DName('TARGET_DRAG_AREA_FT2', attr_text=self.TARGET_DRAG_AREA_FT2))
+
+        names.append(DName('NOM_SA_M2', attr_text=self.NOM_SA_M2))
+        names.append(DName('NOM_SA_FT2', attr_text=self.NOM_SA_FT2))
+
+        names.append(DName('ANEMOMETER_FACTOR', attr_text=self.ANEMOMETER_FACTOR))
+
+        names.append(DName('LOAD_CELL_FACTOR', attr_text=self.LOAD_CELL_FACTOR))
+
+        names.append(DName('TARGET_COEFF_DRAG', attr_text=self.TARGET_COEFF_DRAG))
+
+def f_create_constants_table(target_sheet, loc=None):
```

Comment:
should all of these constants_table values (ie "=B1/0.3048") be based on the constants defined earlier?